Userland/Libraries/LibUnicode/CodeGenerators/GenerateUnicodeData.cpp

```diff
@@ -478,7 +509,14 @@ static constexpr Array<UnicodeData, @code_point_data_size@> s_unicode_data { {)~
         append_field("simple_lowercase_mapping", String::formatted("{:#x}", data.simple_lowercase_mapping.value_or(data.code_point)));
         append_field("simple_titlecase_mapping", String::formatted("{:#x}", data.simple_titlecase_mapping.value_or(data.code_point)));
         append_list_and_size(data.special_casing_indices, "&s_special_casing[{}]"sv);
-        append_list_and_size(data.prop_list, "Property::{}"sv);
+
+        bool first = true;
+        for (auto const& property : data.prop_list) {
+            generator.append(first ? ", " : " | ");
+            generator.append(String::formatted("Property::{}", property));
+            first = false;
+        }
```

Comment:
I'm not a _fan_ of this code, it really looks like something that should use `StringBuilder builder;  builder.join(" | ", data.prop_list)`.

However, with the leading commaa, I don't see any way to actually make it use `join()` without making this code even less pretty to read. There's nothing to do, just wanted to bring attention to it in case you thought of something.

Reply:
It also gets weird because we need to prepend `Property::` to each entry. I *could* add something like `StringBuilder::join_formatted`, where you pass a format string to format each entry with, but I'd rather do that in its own PR.